.circleci/config.yml

```diff
@@ -1,18 +1,17 @@
 version: 2.1
 orbs:
-  ruby: circleci/ruby@0.1.2
+  ruby: circleci/ruby@1.4.0
 
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.6.7
+      - image: cimg/ruby:2.7.5
     executor: ruby/default
     steps:
       - checkout
-      - run:
-          name: Install Bundler
-          command: gem install bundler:2.2
-      - ruby/bundle-install
+      # Install gems with Bundler
+      - ruby/install-deps:
+          with-cache: false
```

Comment:
We should use the cache when possible to speed up CircleCi (not that speed is much of an issue with this repo, but it will help keep things consistent with our other repos). To avoid re-using the wrong cache, we can specify a cache key that will change with either the Gemfile.lock or by manually bumping the `CACHE_VERSION` in CircleCi:
```yaml
      - ruby/install-deps:
          # Need to clear the gem cache? Set or bump the CACHE_VERSION in your
          # CircleCi project: Project Settings > Environment Variables
          key: searchgov_style-{{ .Environment.CACHE_VERSION }}-{{checksum "Gemfile.lock" }}
```
https://support.circleci.com/hc/en-us/articles/115015426888-Clear-project-dependency-cache

Reply:
Good catch 👍